lib/database.js

```diff
@@ -54,6 +54,7 @@ var CrudDriver = function(opts) {
     insert: insert,
     update: update,
     delete: deleteItem,
+    procedure: opts.procedure,
```

Comment:
I have modified the code to expose procedure object if respective database driver has procedure object exists.